README.md

```diff
@@ -164,12 +164,14 @@ truststore|Base64 encoded version of the TLS trust store (in JKS format) used to
 
 ### Resource:  `instaclustr_firewall_rule`                             
 A resource for managing cluster firewall rules on Instaclustr Managed Platform. A firewall rule allows access to your Instaclustr cluster.
+Note: Either rule_cidr OR rule_security_group_id must be provided per rule (but not both)
```

Comment:
[required] could we add an example somewhere showing what an example `rule_security_group_id` would look like? (possibly an indication of where to find it easily in aws as well)

Reply:
Sure - I can add something like "e.g. sg-12345678" ? 
Not sure if we want to go down the rabbithole of explaining where to find it, at this point we are assuming you have already vpc peered and setup your route tables accordingly - it might be an invalid assumption but i would assume if you have come that far you know how to find a security group? 
We might have some documentation i can link to?